src/openforms/config/api/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from .views import PrivacyPolicyInfoView
+
+app_name = "config"
+
+urlpatterns = [
+    path(
+        "privacy_policy_info/",
```

Comment:
```suggestion
        "privacy_policy_info",
```
no trailing slashes in API endpoints